lib/commands/execute.js

```diff
@@ -16,13 +16,15 @@ extensions.execute = async function execute (script, args) {
   return await iosExecute.call(this, script, args);
 };
 
-const iosExecuteAsync = extensions.executeAsync;
-extensions.executeAsync = async function executeAsync (script, args, sessionId) {
+extensions.executeAsync = async function executeAsync (script, args) {
   if (!this.isWebContext()) {
     throw new errors.NotImplementedError();
   }
 
-  return await iosExecuteAsync.call(this, script, args, sessionId);
+  args = this.convertElementsForAtoms(args);
+  this.asyncWaitMs = this.asyncWaitMs || 0;
```

Comment:
should we mutate this value or simply put `this.asyncWaitMs || 0` below?

Reply:
It should be `0` if no one has set it, anyway. If it is not there it is a problem and a mistake.

Further, this is the only place it's used, other that setting it. I'm fine with mutating it.